apps/ui/app/routes/usage/charts/usage-stacked-chart.tsx

```
@@ -0,0 +1,152 @@
+import React, { useMemo } from 'react';
+import { Area, AreaChart, CartesianGrid, XAxis, YAxis } from 'recharts';
+import { format, startOfDay } from 'date-fns';
+import { ChartContainer, ChartTooltip, ChartTooltipContent, ChartLegend, ChartLegendContent, type ChartConfig } from '#components/ui/chart.js';
```

ESLint

[error] Replace `·ChartContainer,·ChartTooltip,·ChartTooltipContent,·ChartLegend,·ChartLegendContent,·type·ChartConfig·` with `⏎··ChartContainer,⏎··ChartTooltip,⏎··ChartTooltipContent,⏎··ChartLegend,⏎··ChartLegendContent,⏎··type·ChartConfig,⏎`

(prettier/prettier)